.github/workflows/tests.yml

```diff
@@ -19,10 +19,17 @@ jobs:
         uses: actions/setup-go@v4
         with:
           go-version: "1.21"
+      - name: Install Task
+        uses: arduino/setup-task@v1
+        with:
+          version: 3.x
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
       - name: Run Tests
         run: |-
-          go test -race -coverprofile=coverage.out -covermode=atomic -v ./...
-          cat coverage.out | grep -v "/enum.go" > coverage.txt
+          task setup
+          task lint
+          task test
+          grep -v "/enum.go" coverage.out > coverage.txt
```

Comment:
any reason to no bake this into `task test` ?

Reply:
Not really :)
I can add it in